### ISSUE TITLE
Accordion Accessibility (Aria-Controls)

### ISSUE THREAD

**Issue by arthur154** [ng-bootstrap/ng-bootstrap#2991]
### Bug description:
The "aria-controls" value is always set on the header. However, the tab panel that it controls does not exist when it is closed, so the aria-control does not have a corresponding ID in the DOM. If you add an else for the closed state of the tab panel and have that create a hidden element with the correct ID, then the issue should be resolved.

### Link to minimally-working StackBlitz that reproduces the issue:
This issue can be seen on the example accordion page (https://ng-bootstrap.github.io/#/components/accordion/examples).

### Versions of Angular, ng-bootstrap and Bootstrap:
Angular: 7.0.4
ng-bootstrap: 4.0.2
Bootstrap: 4.0.0

**Comment by maxokorokov** [comment]
FYI, the workaround for you would be always keeping tab content in DOM with `[destroyOnHide]="false"`. There are no issues with that and we probably should make it the default behaviour, but it would be a breaking change.

I don't see the point in generating a **new** empty element with the correct ID, because it kind of kills the whole point of `aria-controls` → navigating to the controlled element.

I think the easiest solution to fix the inconsistency would be to remove `aria-controls` when the controlled panel is not in DOM to avoid referencing something non-existent.

Another option would be to remove the inner `<div class="card-body">` and not the outer `<div id="..." role="tabpanel">`. This should be tested to see if screen readers will behave correctly.

There is an interesting discussion about the utility of `aria-controls` here → https://github.com/w3c/aria/issues/531

@arthur154 any specific real-life issues/use cases because of this ?

**Comment by IAfanasovMob** [comment]
> I think the easiest solution to fix the inconsistency would be to remove `aria-controls` when the controlled panel is not in DOM to avoid referencing something non-existent.

The [ARIA practices](https://www.w3.org/TR/wai-aria-practices-1.1/#accordion) does not leave the opportunity to remove the `aria-controls` attribute. 

```
The accordion header button element has `aria-controls` set to the ID of the element containing the accordion panel content.
```

**Comment by marthurIntersys** [comment]
@maxokorokov sorry, somehow I missed your response. I think the ask here would be one of two options for the line https://github.com/ng-bootstrap/ng-bootstrap/blob/141683a10ccae4acdb86befe0dc701b79bbc0af9/src/accordion/accordion.ts#L172

1) Remove the ngIf here (the panel should automatically hide when the "show" class is removed)
2) Add a corresponding else to the ngIf that would render an empty/hidden panel that would probably have aria-hidden="true" or an aria-label of "panel X closed"

We did have an issue with Jaws. When you click on a closed tab, there is no feedback and nothing happens.

**Comment by maxokorokov** [comment]
This should be fixed in `14.1.0` with the new accordion directives (#3898, #4437)
